packages/website/docs/RadioButtons.md

```diff
@@ -1,28 +1,52 @@
-## Guidelines
+# Best Practices
 
-(See also: **Guidelines for All Form Controls** - _coming soon!_)
+Use radio buttons to have users select only one option from a short list.
 
-### General
+Radio buttons always come in a set of 2 or more options.
 
--   The set of options should always have a title.
--   Use wording that indicates that the user can **only select one option**. Some users may not know the difference between radio buttons and checkboxes.
--   Put the most common options at the top of the list, unless your options have a strong logical ordering (such as prices or sizes).
--   Keep the number of options per set under 7.
+To present a binary choice (e.g., on/off), do not use radio buttons. Consider using a stand-alone checkbox instead.
```

Comment:
```suggestion
To present a binary choice (e.g., on/off), do not use radio buttons. Consider using a [Toggle](#/form/FlatSelect) instead.
```
Just to avoid contradiction with the Related Components section